apollo-mockserver/src/appleMain/kotlin/com/apollographql/apollo3/mockserver/Socket.kt

```diff
@@ -134,7 +134,7 @@ class Socket(
           try {
             mockServerHandler.handle(request)
           } catch (e: Exception) {
-            MockResponse("MockServerHandler.handle() threw an exception: ${e.message}", 500)
+            throw Exception("MockServerHandler.handle() threw an exception: ${e.message}", e)
```

Comment:
Why not let the original exception bubble up?

Reply:
I was thinking it would make the message a bit more predictable/less dependent on a handler

Reply:
Fair enough 👍